addons/website/static/tests/tours/link_tools.js

```diff
@@ -106,7 +106,7 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
     {
         content: "Set URL.",
         trigger: '.o_we_customize_panel we-input:contains("Your URL") input',
-        run: "edit odoo.com && blur (we-title:contains(Your URL))",
+        run: "edit odoo.com && click(we-title:contains(Your URL))",
```

Comment:
`&& click` to remove? next step is clicking on another img, it should be enough

Reply:
I just modify the blur typo by click, I haven't added any action